pytest_html/plugin.py

```diff
@@ -263,7 +261,8 @@ def append_log_html(self, report, additional_html):
             additional_html.append(log)
 
     def _appendrow(self, outcome, report):
-        result = self.TestResult(outcome, report, self.logfile, self.config)
+        result = self.TestResult(outcome, report, self.logfile, self.config,
+                                 self.rerun)
```

Comment:
`self.rerun` will only be present if the pytest-rerunfailures plugin is installed, if it's not installed I would expect this to raise an exception due to the attribute not being available